### DISCUSSION TITLE
How to add a new attribute to the general setting of a realm?

### DISCUSSION THREAD

**Discussion by ruszajhj** [keycloak/keycloak#17251] [Q&A]
I need to implement an "SPI" where its behavior varies depending on a realm attribute (which I want to add).
This variable must be able to modify its value dynamically from the interface.
So I have two doubts. The first is if new attributes can be added to the realm, since I tried using the API method (PUT /{realm}) sending a Map of new attributes and it gives me an error.
The second would be if in case of being able to add this, there is the possibility of keycloak detecting my new attribute and displaying it in one of the Realm's configuration screens.

**Comment by fredericoschardong** [comment]
We share the same intent, @ruszajhj. I have not found a way to implement this.
